props/saber_fett263_buttons.h

```diff
@@ -381,160 +1755,2779 @@ SaberFett263Buttons() : PropBase() {}
+            wav_player->PlayOnce(&SFX_mnum, 0.0);
+            break;
+          case MENU_BLADE_SETTING:
+            if (blade_num_ == NUM_BLADES) blade_num_ = 0;
+            blade_num_ += 1;
+            BladePreview(blade_num_);
+            SFX_mnum.Select(blade_num_ - 1); 
+            wav_player->PlayOnce(&SFX_mnum, 0.0);
+            break;
+          case MENU_BLADE_COPY:
+            copy_blade_ = (copy_blade_ + 1) % NUM_BLADES + 1;
+            if (copy_blade_ == 0) copy_blade_ = (copy_blade_ + 1) % NUM_BLADES + 1;
+            if (copy_blade_ == blade_num_) copy_blade_ = (copy_blade_ + 1) % NUM_BLADES + 1;
+            BladePreview(copy_blade_);
+            SFX_mnum.Select(copy_blade_ - 1); 
+            wav_player->PlayOnce(&SFX_mnum, 0.0);
+            break;
+            return true;           
+          case MENU_BLADE_LENGTH:
+            if (blade_num_ == NUM_BLADES) blade_num_ = 0;
+            blade_num_ += 1;
+            BladePreview(blade_num_);
+            SFX_mnum.Select(blade_num_ - 1); 
+            wav_player->PlayOnce(&SFX_mnum, 0.0);
+            break;
+          case MENU_COPY_COLOR:
+            choice_ = true;
+            PlayMenuSound("mconfirm.wav");
+            return true;
+#endif
+          case MENU_STYLE:
+            style_num_ = (style_num_ + 1)% num_presets_;
+            char style_arg[10];
+            itoa(style_num_, style_arg, 10);
+            current_preset_.SetStyle(blade_num_, style_parser.SetArgument(current_preset_.GetStyle(blade_num_), 1, style_arg)); 
+            current_preset_.Save();
+            PlayMenuSound("mup.wav");
+            UpdateStyle(current_preset_.preset_num);
+            return true;
+          case MENU_COLOR:
+            return true;
+          case MENU_COLOR_SUB:
+#if NUM_BLADES == 1
+            menu_sub_pos_ = (menu_sub_pos_ + 1) % 3;
+            if (menu_sub_pos_ == 0) menu_sub_pos_ = 1;
+            switch (menu_sub_pos_) {
+              case 1:
+                PlayMenuSound("mcolorop.wav");
+                break;
+              case 2:
+                PlayMenuSound("mresetc.wav");
+                break;
+            }
+#else
+            menu_sub_pos_ = (menu_sub_pos_ + 1) % 4;
+            if (menu_sub_pos_ == 0) menu_sub_pos_ = 1;
+            switch (menu_sub_pos_) {
+              case 1:
+                PlayMenuSound("mcolorop.wav");
+                break;
+              case 2:
+                PlayMenuSound("mcopyc.wav");
+                break;
+              case 3:
+                PlayMenuSound("mresetc.wav");
+                break;
+            }
+#endif
+            return true;
+          case MENU_EFFECT:
+            while (true) {
+              effect_num_ = (effect_num_ + 1)% COLOR_SUB;
+              if (style_parser.UsesArgument(current_preset_.GetStyle(blade_num_), effect_num_ + 2)) break;
+            }
+            if (effect_num_ == 0) effect_num_ = 1;
+            switch (effect_num_) {
+              case 1:
```

Comment:
enum ?

Reply:
the effect_num_ itself is the arg value so made it easier to keep track this way

Reply:
included the ARG enums in last commit